doc/getting-started.adoc

```diff
@@ -83,14 +83,34 @@ $ git branch -vv
 
 === Configure Jenkins
 
-You should have a Jenkins agent already created.
+Jenkins after the first startup of Jenkins container you will have to finish the install in your browser.
+
+* Go to http://localhost:8888/ 
+* fill the (generated) admin password which is available in jenkins startup log
+* choose to select the Jenkins Plugins you want to install
+* unselect ALL plugins from the plugin install selection screen
+
+You should have a Jenkins agent already created but you need need to start it
```

Comment:
```suggestion
You should have a Jenkins agent already created but you need need to start it
```
You should have a Jenkins agent already created but you need to start it

Reply:
Created = configured / declared in Jenkins
Started = ... you need to start it on your host machine with JavaWebStart 😄